### DISCUSSION TITLE
Import liked music/playlists from other platforms (such as Spotify?)

### DISCUSSION THREAD

**Discussion by carsnwd** [nukeop/nuclear#1191] [General]
I have some like 3,000 songs liked on Spotify and was looking at alternatives to switch to. Is there a good script or way to migrate all my liked tracks or playlists to Nuclear?

**Comment by nukeop** [comment]
Here's the relevant documentation:
https://nukeop.gitbook.io/nuclear/user-manual/importing-playlists

**Reply by carsnwd** [reply]
Thank you!
